rest_framework/fields.py

```diff
@@ -1105,7 +1106,10 @@ def enforce_timezone(self, value):
         field_timezone = getattr(self, 'timezone', self.default_timezone())
 
         if (field_timezone is not None) and not timezone.is_aware(value):
-            return timezone.make_aware(value, field_timezone)
+            try:
+                return timezone.make_aware(value, field_timezone)
+            except Exception:
```

Comment:
compat module is in the rest_framework directory.
It should either import the pytz exceptions if available or mock them to avoid breaking code.
And I don't think you already have the `AttributeError` when pytz is not installed do you ?